bundles/org.openhab.binding.deconz/src/main/java/org/openhab/binding/deconz/internal/handler/SensorBaseThingHandler.java

```diff
@@ -106,17 +93,16 @@ public void handleConfigurationUpdate(Map<String, Object> configurationParameter
 
     @Override
     protected void processStateResponse(DeconzBaseMessage stateResponse) {
-        if (!(stateResponse instanceof SensorMessage)) {
+        if (!(stateResponse instanceof SensorMessage sensorMessage)) {
             return;
         }
 
-        SensorMessage sensorMessage = (SensorMessage) stateResponse;
         sensorConfig = Objects.requireNonNullElse(sensorMessage.config, new SensorConfig());
         sensorState = Objects.requireNonNullElse(sensorMessage.state, new SensorState());
 
         // Add some information about the sensor
         if (!sensorConfig.reachable) {
-            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.NONE, "@text/offline.sensor-not-reachable");
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.GONE, "Not reachable");
```

Comment:
This reverts #12245. Why?

Reply:
Regarding the i18n: That's a mistake. The `GONE/NONE`: Debatable. I have never seen OFFLINE when the device was still functional. In most cases it needed to be re-paired to deconz or battery was empty. If it needs to be re-paired it fulfills the definition of `GONE`, in the case of an empty battery it's probably more `NONE`. If you want, I can change it back to `NONE`, although I'm not convinced this is correct.

Reply:
That was not my experience, see #12244.